frontend/src/store/poap/actions.js

```diff
@@ -6,7 +6,7 @@ const jsonFetch = (url) => fetch(url).then((res) => res.json());
  * @notice GET list of all POAP events from the POAP API
  */
 export async function getEvents({ commit }) {
-  const events = await jsonFetch('https://api.poap.xyz/events');
+  const events = await jsonFetch('https://api-event.poap.xyz');
```

Comment:
same, add to technical debt

Reply:
https://poap-devs.atlassian.net/browse/FR-489